src/pages/Licenses/Licenses.tsx

```
@@ -27,7 +27,14 @@ export const LicensesPage = () => {
 
   const { data, isLoading } = useQuery<LicensesSbom>({
     queryKey: ["sbom", tab],
-    queryFn: () => fetch(sbomUrlMap[tab]).then((res) => res.json()),
+    queryFn: async () => {
+      const res = await fetch(sbomUrlMap[tab]);
+      const contentType = res.headers.get("content-type");
+      if (contentType && contentType.includes("application/json")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)